workflows/amr/run.wdl

```diff
@@ -614,13 +614,13 @@ task MakeGeneCoverage {
 task tsvToSam {
     input {
         File contigs
-        File comprehensive_AMR_metrics
+        File final_summary
         String docker_image_id
     }
     command <<<
         set -euxo pipefail
-        PATH_OUTPUT_SAM="test.sam"
-        PATH_OUTPUT_BAM="test.bam"
+        PATH_OUTPUT_SAM="contig_amr_report.sam"
```

Comment:
Made up this name based on `sr_amr_report.sorted.length_100.bam`, but feel free to change